packages/kit/src/views/FirmwareUpdate/componentsV2/FirmwareUpdateProgressBarV2.tsx

```
@@ -0,0 +1,513 @@
+  const [showDebugInfo, setShowDebugInfo] = useState(false);
+  const debugInfo = useMemo(() => {
+    if (process.env.NODE_ENV !== 'production') {
+      return (
+        <Stack my="$6">
+          <Button
+            size="small"
+            onPress={() => {
+              setShowDebugInfo((v) => !v);
+            }}
+          >
+            ProgressDebugInfo ({parseInt(progress.toFixed(), 10)}%)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)